packages/extension-ui/src/Popup/Signing/Tooltip.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ThemeProps } from '../../types';
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+
+interface Props extends ThemeProps {
+  className?: string;
+  content: string;
+  children: React.ReactNode;
+}
+
+const Tooltip: React.FC<Props> = ({ children, className, content }) => {
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  const handleMouseEnter = useCallback(() => {
+    setShowTooltip(true);
+  }, []);
+
+  const handleMouseLeave = useCallback(() => {
+    setShowTooltip(false);
+  }, []);
```

Comment:
Maybe you could use onMouseOver here to avoid those two methods? Just a proposal - nothing major if time's the issue.